surveys/2021-annual-survey/questions.md

```diff
@@ -1,19 +1,138 @@
 # Survey questions
 
-## About you
+### Do you use Rust?
 
-See [who](./design/who.md).
+Select one:
 
-The following are primarily for cohort analysis, secondarily for understanding the shape of the community.
+- Yes, I have used for Rust in the past three months
```

Comment:
We also likely want to base control flow of the survey on this question. Not sure yet how to capture that.

Reply:
Just noting that there seems to be an extra "for" here.